data/mods/Aftershock/npcs/sadie.json

```diff
@@ -0,0 +1,235 @@
+        },
+        "  ",
+        {
+          "npc_at_om_location": "field",
+          "no": "",
+          "yes": "This place is so boring.  We should take people and kill some drugs."
+        },
+        {
+          "npc_at_om_location": "water",
+          "no": "",
+          "yes": "Water water everywhere.  Let's go skinny dipping.  I bet you'll attract some leeches.  I need them for something I want to do to you.  It'll be so weird."
+        },
+        {
+          "npc_at_om_location": "subterranean",
+          "no": "",
+          "yes": "I like it down here.  This reminds me of that time I was down in a place like this and someone creeped up behind me and hit me in the back of the head with a shovel.  *shiver*  It still gives me goosebumps."
+        },
+        {
+          "npc_at_om_location": "forest",
+          "no": "",
+          "yes": "I love nature.  It smells so good out here, like living things growing and multiplying and dying and returning from the dead with horrifying new powers and trying to slowly rip us apart and eat us.  It brings back really happy memories for me."
+        },
+        {
+          "npc_at_om_location": "swamp",
+          "no": "",
+          "yes": "I love swampy places like this.  The smell of the pussywillows, the sound of the crickets, the giant mutant mosquitoes hungry for our blood.  It's like we're in a sexy vampire romance novel."
+        }
+      ]
+    },
+    "responses": [ { "text": "I feel the same Sadie.", "topic": "TALK_Sadie_1" } ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_sadie_injection",
+    "dynamic_line": "I hope you like it, it's one of my favorite things in the world, actually a lot of them all blended together.  It should feel really good.  Or really bad, I guess I don't know much about your body chemistry.  You're still alive, that's a really good sign!",
+    "responses": [
+      {
+        "text": "Wonderful.  Please don't inject me with anything else unless I ask for it in the future.",
+        "effect": [
+          { "u_lose_var": "sadie_injection", "type": "general", "context": "injection" },
+          { "u_add_var": "sadie_heart", "type": "general", "context": "heart", "value": "yes" }
+        ],
+        "topic": "TALK_Sadie_1"
+      }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_sadie_injection2",
+    "dynamic_line": "I'm so glad you liked it.  It really means a lot to me that we have similar body chemistry.  That's so important for all kinds of biological and emotional compatibility for example organ transplants and the reproductive act.  This shows how I feel about you, I think.",
+    "responses": [
+      {
+        "text": "I'm glad we get along so well Sadie.",
+        "effect": [
+          { "u_add_effect": "give_aid", "duration": 2000 },
+          { "u_add_effect": "morale_chat", "duration": 2000 },
+          { "u_add_effect": "sadie_injection", "duration": 2000 }
```

Comment:
Does the duration here use moves or seconds?   I thought it used moves which would mean this effect only lasts twenty seconds.  But I'd need to test it to be certain.

Reply:
The NPC doc says “turns”

Reply:
If it helps, I built a similar thing for Alonso in the refugee center and his duration is 300    `   "effect": [ { "u_add_effect": "al_relaxed", "duration": 300 }, { "u_consume_item": "condom" } ]
`

Reply:
I wanted to be a little more generous here because this is Aftershock, it should be a little extra

Reply:
You were correct, it was gone instantly. Adding two zeros.